identity/app/views/consentJourney.scala.html

```diff
@@ -111,12 +111,13 @@ <h2 class="identity-title">Your existing newsletters</h2>
                 }
                 <div class="manage-account__switches">
                     <ul>
-                    @onlySubscribedToNewsletters.zipWithRowInfo.map { case (newsletter, row) =>
+                    @onlySubscribedToV1Newsletters.zipWithRowInfo.map { case (newsletter, row) =>
                     <li>
                         @fragments.newsletterSwitch(
                             emailPrefsForm,
                             emailSubscriptions,
-                            newsletter = newsletter
+                            newsletter = newsletter,
+                            unchecked = true
```

Comment:
I don't like the switch handling this, wouldn't it be better for `onlySubscribedToV1Newsletters ` to already return the list of V2s, which will be unchecked?

Reply:
Lets do the refactoring of newsletter identity-model after launch.